build-system/corekit/publishing.gradle

```diff
@@ -1,5 +1,6 @@
 apply plugin: 'com.jfrog.bintray'
 apply plugin: 'maven-publish'
+apply plugin: 'signing'
```

Comment:
Sorry I am not sure about what the signing do, is this required for uploading to maven? will other dev able to use the signed sdk library?

Reply:
yes, signing is required by maven, others can do but need to register to maven account. you guys need to register and set up the credential, will share how it's works later.